.env.development

```diff
@@ -1,3 +1,4 @@
+APP_ID='admin-portal'
```

Comment:
`APP_ID` is normally provided by `frontends/frontend-app-admin-portal/*_config.yml` in stage/prod, but in development the only way to supply it currently is via .env.development.